app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
```

Comment:
Code Style Issue: It's recommended to use more descriptive variable names. Instead of `parts`, you might want to use `command_parts` or `command_components` to make it more clear what this variable is for.